isotovideo

```diff
@@ -296,6 +296,9 @@ $testfd->write("GO\n");
 my $interactive = 0;
 my $needinput   = 0;
 
+# screen checking interval in s (float)
+my $interval = 1.;
```

Comment:
can't get it to work. somehow if I try to forward variables where used it's not picked up for the actual check or something. Is the global variable visible by some call that goes out and comes back over RPC which does not work when forwarding the variables?